pkg/build/builder/dockerutil.go

```diff
@@ -2,11 +2,21 @@ package builder
 
 import (
 	"os"
+	"time"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/golang/glog"
 	"github.com/openshift/source-to-image/pkg/tar"
 )
 
+var (
+	// Number of retries of pushing the built Docker image into configured
+	// repository
+	DefaultPushRetryCount = 2
```

Comment:
But not for now.

----- Original Message -----
> >  	"github.com/openshift/source-to-image/pkg/tar"
> >  )
> >  
> > +var (
> > +	// Number of retries of pushing the built Docker image into configured
> > +	// repository
> > +	DefaultPushRetryCount = 2
> 
> Longer term, it might be better to make this configurable (via env vars set
> in the pod) and to do retries up to n times with increasing back off each
> time.
> 
> Another option would be to watch the Docker events for push-related events,
> but unfortunately it doesn't look like push starts/finishes are logged at
> all :frowning:
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2623/files#r31424955